src/ape/types/signatures.py

```diff
@@ -7,6 +7,11 @@
 
 from ape.types import AddressType
 
+# Fix 404 in doc link.
```

Comment:
This is maybe the 404 the original ticket was talking about?
Before, we just got the doc from the package it is from (in the EF), but its links didnt work when in ape's docs

This fixes it!